tests/subsys/debug/tracing/CMakeLists.txt

```diff
@@ -2,7 +2,17 @@
 
 cmake_minimum_required(VERSION 3.13.1)
 
+if(BOARD MATCHES "mps2_.*")
```

Comment:
why mps2? This should work with qemu_*. No need for a regex if we are going to target one platform...

Reply:
updated

Reply:
why not qemu_?